test/integration/services/data/test-copy-first-time-claim-data-to-internal.js

```diff
@@ -49,6 +49,24 @@ describe('services/data/copy-first-time-claim-data-to-internal', function () {
           expect(results[0].Status, 'Claim.Status should be NEW').to.be.equal(statusEnum.NEW)
         })
     })
+    .then(function () {
+      return testHelper.deleteAll(reference, 'IntSchema')
```

Comment:
break out the new test case into a new describe and add a `after` to clean up each, only way to be sure clean up will run